### PR TITLE
Added tooltip functionality 

### PR DESCRIPTION
**Pull request by Madhurima94**
**Fixed Issues**
#21962

**Files Changed**
shared.module.ts - Added MD2TooltipModule for implementing tooltip functionality
test-launch-dialog.component.ts-Added code for displaying tooltip message and selecting the test link on clicking the content_copy icon.
test-launch-dialog.html-Binded tooltip message to content_copy icon and added methods for changing tooltip message and selecting the test link.

**Checks**

- [x] Naming Conventions
- [ ] Server side code comments (proper English, grammar and no spelling mistakes)
- [x] Client side code comments (proper English, grammar and no spelling mistakes)
- [x] Unused variables, methods, blank spaces and name spaces. 
- [x] Optimal Code and code formatting
- [x] Commit History is proper, no merge is done. 
- [x] Commit/pull request messages should be proper to justify your feature
- [x] All test cases are executed provided by tester.

---
This change is [Reviewable](https://reviewable.io/reviews/promact/trappist/349)

### CODE REVIEW

File: Trappist/src/Promact.Trappist.Web/wwwroot/app/tests/test-settings/test-launch-dialog.html [hunk 1]
```diff
@@ -7,10 +7,10 @@ <h1 class="pull-left" md-dialog-title>Share Test</h1>
     <div class="form-group">
       <label>Global URL</label>
       <div class="url-container">
-        <input type="text" class="form-control" [(ngModel)]="testLink" name="Test Link" />
-        <em class="material-icons" ngxClipboard [cbContent]="testLink" (cbOnSuccess)="copiedContent = true">content_copy</em>
+        <input type="text" class="form-control" [(ngModel)]="testLink" name="TestLink" id="TestLink" #TestLink onblur="this.select();" />
```
Comment (Krunal-Promact) [inline Trappist/src/Promact.Trappist.Web/wwwroot/app/tests/test-settings/test-launch-dialog.html]: id and name should be camel case

Comment (Krunal-Promact) [inline Trappist/src/Promact.Trappist.Web/wwwroot/app/tests/test-settings/test-launch-dialog.html]: why is onblur required?

Comment (Madhurima94) [inline Trappist/src/Promact.Trappist.Web/wwwroot/app/tests/test-settings/test-launch-dialog.html]: I have used onblur to keep the test link selected after button click else the test link is not remaining selected.

Comment (Madhurima94) [inline Trappist/src/Promact.Trappist.Web/wwwroot/app/tests/test-settings/test-launch-dialog.html]: I am changing the name to Link as testLink is the name of the variable which is being binded to the textarea. On using testLink for id and name again and passing it as parameter, error will occur for similar name reference.

Comment (Krunal-Promact) [inline Trappist/src/Promact.Trappist.Web/wwwroot/app/tests/test-settings/test-launch-dialog.html]: you can give different name in that case.

Comment (Madhurima94) [inline Trappist/src/Promact.Trappist.Web/wwwroot/app/tests/test-settings/test-launch-dialog.html]: Yes, I am changing the name used for "id" and "name" from "TestLink" to "Link" or shall I change the name of the variable from "testLink" to something else?

Comment (dharmeshpipariya) [inline Trappist/src/Promact.Trappist.Web/wwwroot/app/tests/test-settings/test-launch-dialog.html]: onblur is not required here and select method too.

Comment (Madhurima94) [inline Trappist/src/Promact.Trappist.Web/wwwroot/app/tests/test-settings/test-launch-dialog.html]: But the content of the test box that is the test link is not remaining selected on eliminating this event(onblur) and method(select).

**Pull request merged**